src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -57,7 +57,7 @@
         "symfony/workflow": "^4.3|^5.0",
         "symfony/yaml": "^3.4|^4.0|^5.0",
         "symfony/property-info": "^3.4|^4.0|^5.0",
-        "symfony/lock": "^3.4|^4.0|^5.0",
+        "symfony/lock": "^4.4|^5.0",
```

Comment:
Why?

Reply:
I mean, couldn't we inject the right class in the extension?

Reply:
Because KISS :)